### ISSUE TITLE
Debian repo broken since the latest update

### ISSUE THREAD

**Issue by patrickli** [madeye/shadowsocks-org#3]
When running `apt-get update` I get this error:

```
E: GPG error: http://shadowsocks.org wheezy Release: The following signatures were invalid: NODATA 1 NODATA 2
```

**Comment by parsafatehi** [comment]
W: GPG error: http://shadowsocks.org wheezy Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1939D14B1D27208A

W: GPG error: http://shadowsocks.org squeeze Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1939D14B1D27208A

**Comment by madeye** [comment]
@parsafatehi http://shadowsocks.org/en/download/servers.html
